### ISSUE TITLE
is this code still maintened?

### ISSUE THREAD

**Issue by lm17918** [cvqluu/Additive-Margin-Softmax-Loss-Pytorch#18]
I would like to add some improvements to this repo and I just want to know if anyone will review the changes since it doesn t look like there is much going on
